<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lang</class>
 <widget class="QWidget" name="Lang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>192</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>192</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>192</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Language - Langue</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {	background-color: #5A5A5A; }

QPushButton {
	color: #ffffff;
	border: none;
	padding: 5px;
	background-color: #3a3a3a;
	border-radius: 5px;
}

QPushButton::pressed { background-color: #2d5bd4; }</string>
  </property>
  <widget class="QLabel" name="lang_title_lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 97, 167);</string>
   </property>
   <property name="text">
    <string>GENPASS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="lang_sub_title_lbl">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>55</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 97, 167);</string>
   </property>
   <property name="text">
    <string>Ayckinn's Password Generator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose your language / Choisissez votre langue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="en_btn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>English</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fr_btn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fran√ßais</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
